sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/Book.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.ts.reactive;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import io.quarkus.hibernate.reactive.panache.PanacheEntityBase;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@Entity
+@Table(name = "books")
+public class Book extends PanacheEntityBase {
+    private static final int REASONABLY_LONG_TEXT = 100;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, nullable = false)
+    private Integer id;
+
+    @NotNull
+    @Size(max = REASONABLY_LONG_TEXT)
+    private String title;
+
+    @NotNull
+    private Integer author;
+
+    public Book() {
+    }
+
+    public Book(String title) {
+        this.title = title;
+    }
+
+    public static Multi<Book> all() {
+        return streamAll();
+    }
+
+    public static Uni<Book> byId(Integer id) {
+        return findById(id);
+    }
+
+    public static Uni<Book> create(Integer author, String name) {
```

Comment:
Why create is a static method? 

could be part of a Book instance and by this way be able to persist a book.
For example, 

```
 public Uni<Book> create() {
        return this.persistAndFlush();
    }
```

Now imagine that you have a Rest method that persists a book

```
@PUT
    @Path("books/{author}/{name}")
    public Uni<Response> createBook(Integer author, String name) {
 Book book = new Book();
        book.setAuthor(author);
        book.setTitle(name);
         book.create()
```

Reply:
>Why create is a static method
By definition, there is no object to have instance methods before we create one.

In your example, we do not need any "create" method at all, we can just call persistAndFlush directly.

And with static method, your example become much more compact `Book.create(author, name)`